configs/mpc_agent.conf

```diff
@@ -1,6 +1,9 @@
 ######### Carla config #########
 --carla_num_people=50
 --carla_num_vehicles=10
+######### Perception config #########
+--perfect_obstacle_detection=True
```

Comment:
Can you instead use `carla_obstacle_detection` and `carla_traffic_light_detection`? These flags make sure that the MPC agent operator gets data directly from CARLA, whereas the perfect* flags introduce perfect versions of the operators, which you don't really need for your MPC agent operator.

Reply:
Yeah sure